Linphone/Commons/PanelOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BelledonneCommunications.Linphone.Commons
+{
+    public class PanelOptions
```

Comment:
Why this class created? Couldn't we use settings class itself?

Reply:
Its used on ICoreHttpClient to reduce exsiting coupling with linphone sdk.

Reply:
I think this class is overkill, please remove it, and save panel url into settings

for decoupling http client from linphone, please get simple string as url in desired functions, instead of using Microsoft Options

Reply:
Resolved, pls review.